src/Azure.Functions.Cli/Common/Constants.cs

```diff
@@ -99,6 +100,16 @@ public static class KuduLiteDeploymentConstants
             };
         }
 
+        public static class E2ETestConstants
```

Comment:
I'd move this to the test project. I know it's used in a couple of places in this project, but I think we can change that usage to avoid having this here.